packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * Added
   * Added screen-reader support for strikethroughs to convey an entered in error status.
+  * Added additonal translations for strikethrough alt text.
+
+* Fixed
+  * Fixed the clincial result display check for if the unit is valid when reading alt text for strikethroughs.
```

Comment:
Can we reword this to be more consumer-oriented rather than referring to the internal `ClinicalResultDisplay` component?

Reply:
Sure thing, is this better? [b7c893a](https://github.com/cerner/terra-clinical/pull/873/commits/b7c893a81023fec9b20769733b305439b79839e0)